### PR TITLE
feat(rust): Add `macro_invocation` as a type of `@call.outer`.

### PR DESCRIPTION
**Pull request by kylechui**
To my understanding, macro invocations should be considered a type of function call. Would also resolve [this issue](https://github.com/kylechui/nvim-surround/issues/290#issuecomment-1854629989).

### CODE REVIEW

File: queries/rust/textobjects.scm [hunk 1]
```diff
@@ -70,6 +70,7 @@
 (unsafe_block (_)? @block.inner) @block.outer
 
 ;; calls
+(macro_invocation) @call.outer
```
Comment (ribru17) [inline queries/rust/textobjects.scm:73]: Would it be possible to also add the corresponding `@call.inner` group? Maybe something like:

```scheme
(macro_invocation (token_tree . "(" . (_) @_start (_)? @_end . ")"
  (#make-range! "call.inner" @_start @_end)))
```

Comment (kylechui) [reply queries/rust/textobjects.scm:73]: Some basic testing using the query editor seems to validate this code; added in a commit.

**Review by ribru17 — APPROVED** [review]

**Comment by kiyoon** [comment]
Can you rebase it to the master so that the tests pass? It is required to pass before merging and the older master commit was failing. Sorry and thank you!

**Comment by kylechui** [comment]
Woops accidentally merged instead, hopefully that's fine. Sorry!

**Pull request merged**